src/test/java/net/snowflake/client/log/JDK14LoggerConsoleHandlerOverrideLatestIT.java

```diff
@@ -36,4 +39,42 @@ public void shouldOverrideConsoleLogger() throws Exception {
       assertTrue(Arrays.stream(handlers).anyMatch(h -> h instanceof StdOutConsoleHandler));
     }
   }
+
+  /** Added in > 3.20.0 */
+  @Test
+  public void shouldOverrideConsoleLoggerWithSpecificThreshold() throws Exception {
```

Comment:
Maybe we should add some tests that verify this is logging as expected, for example by temporarily changing System.out and System.err

Reply:
I added validation for std out and std err